src/main/java/org/reactome/release/downloaddirectory/GenerateGOAnnotationFile/GOAGeneratorUtilities.java

```diff
@@ -1,102 +1,139 @@
 package org.reactome.release.downloaddirectory.GenerateGOAnnotationFile;
 
-import org.gk.model.ClassAttributeFollowingInstruction;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.gk.model.GKInstance;
-import org.gk.model.InstanceUtilities;
 import org.gk.model.ReactomeJavaConstants;
-import org.gk.schema.SchemaClass;
 
 import static org.reactome.release.downloaddirectory.GenerateGOAnnotationFile.GOAGeneratorConstants.*;
 
 import java.io.*;
 import java.nio.file.*;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
 import java.util.*;
 import java.util.stream.Collectors;
 import java.util.zip.GZIPOutputStream;
 
 public class GOAGeneratorUtilities {
```

Comment:
Usually it's a good idea to create a private no-op constructor to a utilities class, to prevent instantiation.

Reply:
Done!